Source/Libraries/CorruptCore/CorruptCore.cs

```diff
@@ -24,6 +24,15 @@ public class ProblematicProcess
         public string Message { get; set; }
     }
 
+    public class GameClosedEventArgs : EventArgs
```

Comment:
We'll see if this change causes build issues in other projects. If it causes a lot of breaks I may make an exception

Reply:
Seems fine. Not sure if anything is actually fully utilizing these EA at the moment or if they're there for API futureproofing. Might be in one of the plugins that isn't on github at the moment, but we can adapt